python/ray/dashboard/subprocesses/module.py

```diff
@@ -217,4 +220,5 @@ def sigterm_handler(signum, frame):
 
     ray._private.utils.set_sigterm_handler(sigterm_handler)
 
+    loop.run_until_complete(task)
```

Comment:
why this change?

Reply:
Otherwise the task result is not retrieved, and if the task raises an exception, the logs will go to wrong files.